bookstore/clone.py

```diff
@@ -137,12 +139,12 @@ def construct_template_params(self, s3_bucket, s3_object_key):
         else:
             redirect_contents_url = url_path_join(self.base_url, self.default_url)
         clone_api_url = url_path_join(self.base_url, "/api/bookstore/clone")
-        model = {"s3_bucket": s3_bucket, "s3_key": s3_object_key}
+        model = {"s3_bucket": s3_bucket, "s3_key": s3_object_key, "s3_version_id": s3_version_id}
         template_params = {
             "post_model": model,
             "clone_api_url": clone_api_url,
             "redirect_contents_url": redirect_contents_url,
-            "source_description": f"'{s3_object_key}' from the s3 bucket '{s3_bucket}'",
+            "source_description": f"'{s3_object_key}'{' version: '+s3_version_id if s3_version_id else ''} from the s3 bucket '{s3_bucket}'",
```

Comment:
Might be better to move the conditional version text into a variable above to make it more readable

Reply:
Sounds good!

Reply:
this change has been made in the latest version of this PR